azure-wildfly/src/test/java/sunstone/azure/armTemplates/archiveDeploy/webapp/WebAppDeploySuiteTests.java

```diff
@@ -14,5 +14,4 @@
 @Suite
 @SelectClasses({AzureWebAppDeployFirstTest.class, AzureWebAppUndeployedSecondTest.class})
 public class WebAppDeploySuiteTests {
-    public static final String webAppDeployGroup = "deploytestWebApp";
```

Comment:
Tests run within this suite must use same resource group. Please revert this change.

Reply:
I reverted to suite common name, but kept it unique, feel free to close this. I'll mark duplicates of this as resolved